python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -4073,28 +4074,28 @@ def test_empty_dataframe_describe():
 
 
 def test_as_column_types():
-    col = column.as_column(cudf.Series([]))
+    col = column.as_column(_create_cudf_series([]))
```

Comment:
Same here and below -- I'm not sure if I understand why these helper functions are needed. It seems like the test construction might need subtle changes but I don't think this casting helper function is the right answer. It's not announcing what it is actually doing -- the differences between `cudf.Series` and `_create_cudf_series` should be in the name, if this is needed (like `_create_cudf_series_float64_default`).

Reply:
Agreed, reverted the change here and passed explicit dtypes. Renamed the helper to `_create_cudf_series_float64_default` and `_create_pandas_series_float64_default`. These will be removed in `pandas-2.0` upgrade.